src/cosmos/ControlledSwap.fixture.tsx

```diff
@@ -46,6 +46,9 @@ function Fixture() {
 
   const connector = useProvider(SupportedChainId.MAINNET)
 
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
mehh i feel like this makes it harder to read tbh

Reply:
this guarantees that all of the events are there, whereas otherwise there's a lot of manual checking required.
it's just meant to make it easier to test, and it's a fixture, so I'm less worried about breaking the rules (of hooks).